README.md

```diff
@@ -142,12 +146,47 @@ Data resiliency is not paramount though as this dataset can be rebuilt upon your
 
 Terraboard currently supports configuration in three different ways:
 
-1. Environment variables
-2. CLI parameters
-3. Configuration file (YAML). A configuration file example can be found in the root directory of this repository.
+1. Environment variables **(only usable for mono provider configuration)**
+2. CLI parameters **(only usable for mono provider configuration)**
+3. Configuration file (YAML). A configuration file example can be found in the root directory of this repository and in the `test/` subdirectory.
+
+**Important: all flags/environment variables related to the providers settings aren't compatible with multi-provider configuration! For that, you must use the Yaml config file to be able to configure multiples buckets/providers.**
 
 The precedence of configurations is as described below.
 
+### Multiple buckets/providers
+
+In order to be able to link to Terraboard multiples buckets or even providers, you must use the Yaml configuration method.
+
+- Set the environment variable **CONFIG_FILE** or the flag **-c** / **--config-file** to a valid Yaml config file.
+
+- In the Yaml file, specify your desired providers configuration. For example with two S3 buckets:
```

Comment:
```suggestion
- In the YAML file, specify your desired providers configuration. For example for two S3 buckets:
```

Reply:
Don't we need to specify the no versioning and no locks options in this example since it uses minio?